docs/ballerina-by-example/examples/fork-join-variable-access/fork-join-variable-access.bal

```diff
@@ -1,87 +1,83 @@
             s -> fork;
         }
     } join (all) (map results) {
 
         any[] r1;
         any[] r2;
-        // Declare variables to receive the results from forked workers W1 and W2.
-        // The 'results' map contains a map of any type array from each worker
-        // defined within the fork-join statement.
-        // Values received from worker W1 are assigned to any array of r1.
+        // Declare variables to receive the results from the forked workers W1 and W2.
+        // The 'results' map contains a map of the results from the workers within the fork-join statement in the
+        // 'any' type array.
+        // Values received from worker W1 are assigned to the 'any' type array of r1.
         var x1 = <any[]>results["W1"];
         match x1 {
             any[] val  => {r1 = <any[]> val;}
             error e => {io:println(e.message);}
         }
 
-        // Values received from worker W2 are assigned to any array of r2.
+        // Values received from worker W2 are assigned to 'any' type array of r2.
         var x2 = <any[]>results["W2"];
         match x2 {
             any[] val  => {r2 = <any[]> val;}
             error e => {io:println(e.message);}
         }
 
-        // Getting the 0th index of array returned from worker W1.
+        // Get the 0th index of the array returned from worker W1.
         int p;
         p = check <int>r1[0];
-        // Getting the 1th index of array returned from worker W1.
+        // Getting the 1st index of the array returned from worker W1.
         string l;
         var indexL = <string>r1[1];
         match indexL {
             string val  => {l = <string > val;}
         }
 
-        // Getting the 0th index of array returned from worker W2.
+        // Getting the 0th index of the array returned from worker W2.
         string q;
         var indexQ = <string>r2[0];
         match indexQ {
             string val  => {q = <string > val;}
         }
 
-        // Print values received from workers within join block.
+        // Print the values received from workers within the join block.
         io:println("[default worker] within join:
         Value of integer from W1 is [" + p + "]");
         io:println("[default worker] within join:
         Value of string from W1 is [" + l + "]");
         io:println("[default worker] within join:
         Value of string from W2 [" + q + "]");
     }
-    // Print values after the fork-join statement to check effect on variables.
-    // Value type variables have not been changed since they are passed in as a
-    // copy of the original variable.
+    // Print the values after the fork-join statement to check the values of the variables.
     io:println("[default worker] after fork-join:
         Value of integer variable is [" + i + "]
         Value of string variable is [" + s + "]");
-    // Reference type variables are changed since they have passed in as a
```

Comment:
This statement is also shouldn't be removed. This shows the behavior difference from passing by value vs passing by reference.